desc/vmec.py

```diff
@@ -775,9 +788,9 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         x_mn = np.zeros((surfs - 1, m.size))
         for i in range(surfs - 1):
             if eq.sym:
-                x_mn[i, :] = cos_transform.fit(data[i, :])
+                x_mn[i, :] = cosfit(data[i, :])
             else:
-                x_mn[i, :] = full_transform.fit(data[i, :])
+                x_mn[i, :] = fullfit(data[i, :])
         xm, xn, s, c = ptolemy_identity_rev(m, n, x_mn)
         gmnc[0, :] = 0
         gmnc[1:, :] = -c  # negative sign for negative Jacobian
```

Comment:
Do we still think these hard-coded negative signs are needed after this change? I think so, but want to double check.

Reply:
they do seem to be. Flipping the m modes flips the effective definition of theta, but all the quantities are still off by a minus sign, likely because of the jacobian being a factor going into B, J etc.